lib/tools/src/tprof.erl

```diff
@@ -1267,7 +1271,9 @@ disable_patterns(Session, Patterns, Map, Type) ->
 %% ad-hoc profiler implementation
 do_profile(What, Options) ->
     %% start a new tprof server
-    Pid = start_result(start_link(Options#{ session => tprof_profile })),
+    DefaultOptions = #{session => tprof_profile},
```

Comment:
I wonder if we even need to do that, maybe relying on `start_link/1` defaults (default session name is `tprof`) is OK?